Automattic/wp-calypso
# Gutenberg: update npm packages to latest versions

Update Gutenberg npm packages to latest versions.

### Testing instructions
1. Navigate to `/gutenberg/post/{siteSlug}` and verify that everything still works as before.
2. Calypso smoke test.
